src/models/trip.model.ts

```diff
@@ -0,0 +1,34 @@
+import mongoose, { Schema } from 'mongoose';
+import { MissionType } from '../validationSchemas/tripSchema';
+
+const StopSchema = new mongoose.Schema(
+  {
+    location: { lon: Number, lat: Number },
+    address: {
+      street: { type: String, required: false },
+      city: { type: String, required: false },
+      state: { type: String, required: false },
+      zip: { type: String, required: false },
+    },
+    mission: {
+      type: {
+        type: String,
+        enum: Object.values(MissionType),
+      },
+      data: Schema.Types.Mixed,
+    },
+  },
+  {
+    _id: false,
+  }
+);
+
+const TripSchema = new mongoose.Schema({
+  creator: String,
```

Comment:
shouldn't the creator and the guides be the user._id?

Reply:
It should be but for the tests it was easier in the meantime to put a string